examples/custom/variables.tf

```diff
@@ -63,3 +63,15 @@ variable "access_token_expiration" {
   description = "Defines the access token expiration in seconds"
   default     = "3600"
 }
+
+variable "fs_validated" {
+  description = "Enable use of financial services validated products in the account"
+  type        = bool
+  default     = false
+}
+
+variable "user_list_visibility" {
+  description = "Enable restriction of user list visibility in the account"
+  type        = bool
+  default     = false
+}
```

Comment:
In the custom example, we DO want these set to false, however can you update the example so these are not exposed as variables, but are hard coded in main.tf? Can you do it for the other variables too please? We don't want to expose all these as variables in an example. Thanks